package.json

```diff
@@ -22,6 +22,7 @@
     "apollo-server-express": "2.5.0",
     "argparse": "1.0.10",
     "async": "2.6.1",
+    "aws-sdk": "^2.460.0",
```

Comment:
We use fixed dependencies in this repository.

Reply:
ahh, completely slipped my mind, will commit the correction. Would it be worth putting this as a lint rule or something in the future? More than happy to create an issue and look into it when I can.

Reply:
I don't know something simple that would do the trick. Not sure it's worth it if it's complicated.

Proposals welcome!

Reply:
Alright, I'll have a look around soon and will create a new issue & let you know if I find something worthwhile.